controllers/api/favorite-pwa.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2015-2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+const router = express.Router(); // eslint-disable-line new-cap
+
+const verifyIdToken = require('../../lib/verify-id-token');
+const libFavoritePwa = require('../../lib/favorite-pwa');
+const FavoritePwa = require('../../models/favorite-pwa');
+const User = require('../../models/user');
+
+/**
+ * GET /favorite-pwa/
+ *
+ * Returns all Favorite PWAs for a user
+ */
+router.get('/', (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+
+  const idToken = req.query.idToken;
```

Comment:
We send it as part of the form data in Add PWA.
How would the client send it in this case? Do I get id_token as part of the Authorization header?